ligo/Makefile

```diff
@@ -0,0 +1,32 @@
+# SPDX-FileCopyrightText: 2020 TQ Tezos
+# SPDX-License-Identifier: MIT
+
+# Ligo executable
+LIGO ?= ligo
+
+# Compile code
+BUILD = $(LIGO) compile-contract --syntax cameligo
+
+# Where to put build files
+OUT ?= out
+
+.PHONY: all trivialDAO build clean clean_all
+
+all: build
+
+# Compile LIGO contract into its michelson representation.
+build: src
+	mkdir -p $(OUT)
+	# ============== Compiling contract ============== #
+	$(BUILD) src/base_DAO.mligo base_DAO_contract --output-file $(OUT)/baseDAO.tz
+	# ============== Compilation successful ============== #
+	# See "$(OUT)/baseDAO.tz" for compilation result #
+
+	# strip the surrounding braces and indentation,
+	# note that dollar char is escaped as part of Makefile
```

Comment:
Why are we doing it?

Reply:
LIGO prints a contract as 

```
{ parameter ...;
  storage ...;
  code ...; }
```

This makes the contract more difficult to read (for instance I needed to do `diff baseDAOLorentz.tz baseDAOLigo.tz`, and it got broken because of the braces). Since I needed to implement this, probably it makes sense to leave there (though I don't like these extra lines being printed on each `make build`, not sure what to do with this).